decentraland/helpers/index.js

```diff
@@ -81,40 +82,44 @@ export const addressesAreEqual = (address1, address2) => {
   return address1.toUpperCase() === address2.toUpperCase();
 };
 
-export const approveManaSpending = async fromAccount => {
+export const approveManaSpending = fromAccount => {
   const contracts = getContracts();
   const { manaContract, marketplaceContract } = contracts;
   const toAddress = marketplaceContract.getAddress();
   manaContract.setWallet(fromAccount);
+
   const action = manaContract.approve(toAddress, HALF_MILLION);
-  await action.waitForNonceToUpdate();
+  return action;
 };
 
-export const fundAccountWithEthers = async accountAddress => {
+export const fundAccountWithEthers = accountAddress => {
```

Comment:
Any helpers in this file that seem general are good candidates for moving into the SDK as features

Reply:
Feel free to resolve this

Reply:
Ok, Resolving this.